rest-api/test/unit_test/dao_test/participant_dao_test.py

```diff
@@ -271,6 +271,9 @@ def test_pairs_unset(self):
     self.assertEquals(paired.hpoId, self._test_db.hpo_id)
     self.assertEquals(paired.providerLink, make_primary_provider_link_for_id(self._test_db.hpo_id))
     self.assertEquals(self.participant_summary_dao.get(participant_id).hpoId, self._test_db.hpo_id)
+    self.assertEquals(paired.organizationId, self._test_db.organization_id)
```

Comment:
It'd be good to have some tests somewhere else in here that check that pairing at the awardee, organization, and site level (via dao.update()) works as intended in terms of setting these fields.

(You could for example copy  test_update_right_expected_version but instead of updating providerLink, update hpoId, organizationId, or siteId.)